host-controller/src/main/java/org/jboss/as/host/controller/HostControllerEnvironment.java

```diff
@@ -496,13 +500,29 @@ public HostControllerEnvironment(Map<String, String> hostSystemProperties, boole
         // Note the java.security.manager property shouldn't be set, but we'll check to ensure the security manager should be enabled
         this.securityManagerEnabled = securityManagerEnabled || isJavaSecurityManagerConfigured(hostSystemProperties);
         this.processType = processType;
+
+        this.stream = getEnumProperty(hostSystemProperties, FEATURE_STREAM, FeatureStream.DEFAULT);
+        WildFlySecurityManager.setPropertyPrivileged(FEATURE_STREAM, this.stream.toString());
```

Comment:
Is not this somehow redundant? `hostSystemProperties` should already contain the properties for this process. Maybe you want it explicitly when it got the default value?

Reply:
Yes - you are right.  I'll fix this to call WildFlySecurityManager.setPropertyPrivileged(...) conditionally.

Reply:
Looks good, but I was expecting something like this:

```
if(hostSystemProperties.get(FEATURE_STREAM) == null) {
    WildFlySecurityManager.setPropertyPrivileged(...) 
}
```

So we set it with the default value only if it was not set before.